client/functions.lua

```diff
@@ -85,28 +85,43 @@ RegisterNUICallback('getNotifyConfig', function(_, cb)
     cb(QBCore.Config.Notify)
 end)
 
-function QBCore.Functions.Notify(text, texttype, length)
+
+---@alias NotificationPosition 'top' | 'top-right' | 'top-left' | 'bottom' | 'bottom-right' | 'bottom-left'
+---@alias NotificationType 'inform' | 'error' | 'success'
+---@alias DeprecatedNotificationType 'primary'
+
+---@class NotifyProps
+---@field id? string notifications with the same id will not be on the screen at the same time
+---@field title? string displayed to the player
+---@field description? string displayed to the player
+---@field duration? number milliseconds notification is on screen
+---@field position? NotificationPosition
+---@field type? NotificationType
+---@field icon? string https://fontawesome.com icon name
+---@field iconColor? string css color value for the icon
+
+---Text box popup for player which dissappears after a set time.
+---@param props NotifyProps
+function QBCore.Functions.NotifyV2(props)
+    props.style = nil
+    lib.notify(props)
```

Comment:
Not sure how to prevent lib from being a lint failure. I was looking at qb-weed as an example and didn't see anything being done different.

Reply:
what if you try to do local lib = lib on top?

Reply:
The linter doen't support ox_lib at the moment, just ignore it.